M/mlpack/build_tarballs.jl

```diff
@@ -137,6 +137,7 @@ products = [
     LibraryProduct("libmlpack_julia_lars", :libmlpack_julia_lars),
     LibraryProduct("libmlpack_julia_linear_regression",
         :libmlpack_julia_linear_regression),
+    LibraryProduct("libmlpack_julia_linear_svm", :libmlpack_julia_linear_svm)
```

Comment:
Missing comma:
```suggestion
    LibraryProduct("libmlpack_julia_linear_svm", :libmlpack_julia_linear_svm),
```
:wink: